src/ui/macstr_ui.py

```diff
@@ -0,0 +1,201 @@
+        # These are the radio buttons
+        # -- this will be used to select and edit the JSON file that will dictate
+        # -- which will be considered as striation and non-striation
+        # -- a function will be executed indicating if the image has striation patterns
+        # -- or not
+        # ------------------------------
+        self.radioButton = QtWidgets.QRadioButton(self.centralwidget)
+        self.radioButton.setGeometry(QtCore.QRect(30, 210, 106, 23))
+        self.radioButton.setObjectName("radioButton")
+
+        self.radioButton_2 = QtWidgets.QRadioButton(self.centralwidget)
+        self.radioButton_2.setGeometry(QtCore.QRect(140, 210, 106, 23))
+        self.radioButton_2.setObjectName("radioButton_2")
+
+        self.radioButton_3 = QtWidgets.QRadioButton(self.centralwidget)
+        self.radioButton_3.setGeometry(QtCore.QRect(390, 210, 106, 23))
+        self.radioButton_3.setObjectName("radioButton_3")
+
+        self.radioButton_4 = QtWidgets.QRadioButton(self.centralwidget)
+        self.radioButton_4.setGeometry(QtCore.QRect(280, 210, 106, 23))
+        self.radioButton_4.setObjectName("radioButton_4")
+
+        self.radioButton_5 = QtWidgets.QRadioButton(self.centralwidget)
+        self.radioButton_5.setGeometry(QtCore.QRect(640, 210, 106, 23))
+        self.radioButton_5.setObjectName("radioButton_5")
+
+        self.radioButton_6 = QtWidgets.QRadioButton(self.centralwidget)
+        self.radioButton_6.setGeometry(QtCore.QRect(530, 210, 106, 23))
+        self.radioButton_6.setObjectName("radioButton_6")
+
+        self.radioButton_7 = QtWidgets.QRadioButton(self.centralwidget)
+        self.radioButton_7.setGeometry(QtCore.QRect(140, 460, 106, 23))
+        self.radioButton_7.setObjectName("radioButton_7")
+
+        self.radioButton_8 = QtWidgets.QRadioButton(self.centralwidget)
+        self.radioButton_8.setGeometry(QtCore.QRect(30, 460, 106, 23))
+        self.radioButton_8.setObjectName("radioButton_8")
+
+        self.radioButton_9 = QtWidgets.QRadioButton(self.centralwidget)
+        self.radioButton_9.setGeometry(QtCore.QRect(390, 460, 106, 23))
+        self.radioButton_9.setObjectName("radioButton_9")
+
+        self.radioButton_10 = QtWidgets.QRadioButton(self.centralwidget)
+        self.radioButton_10.setGeometry(QtCore.QRect(280, 460, 106, 23))
+        self.radioButton_10.setObjectName("radioButton_10")
+
+        self.radioButton_11 = QtWidgets.QRadioButton(self.centralwidget)
+        self.radioButton_11.setGeometry(QtCore.QRect(640, 460, 106, 23))
+        self.radioButton_11.setObjectName("radioButton_11")
+
+        self.radioButton_12 = QtWidgets.QRadioButton(self.centralwidget)
+        self.radioButton_12.setGeometry(QtCore.QRect(530, 460, 106, 23))
+        self.radioButton_12.setObjectName("radioButton_12")
+
+        # ------------------------------
+        # Refresh button
+        # -- used to reload the images
+        # ------------------------------
```

Comment:
In addition to a button, could this application automatically detect changes in a relevant location for refreshing images (perhaps providing a prompt)?

Reply:
Changes will be reflected in the loaded `image_cords.json` file. When the user select a positive case for striation, it will trigger an event updated the `.json` file.